feature-detects/img/loading.js

```diff
@@ -0,0 +1,19 @@
+/*!
+{
+  "name": "image native lazy loading",
+  "property": "loading",
```

Comment:
I'd like to use "lazyloading" for this (which you also use in the actual test down below)